src/main/java/com/mide/gangsaeng/board/BoardReadService.java

```diff
@@ -0,0 +1,5 @@
+package com.mide.gangsaeng.board;
+
+public interface BoardReadService {
```

Comment:
Why did you create new `BoardReadService`?

Reply:
I only want to override the `read()` method.